lib/kafo/puppet_command.rb

```diff
@@ -36,11 +36,26 @@ def append(suffix)
 
     def self.search_puppet_path(bin_name)
       bin_path = (::ENV['PATH'].split(File::PATH_SEPARATOR) + ['/opt/puppetlabs/bin']).find do |path|
-        File.executable?(File.join(path, bin_name))
+        File.executable?(File.join(path, 'puppet'))
```

Comment:
That's actually not as ugly as I thought, but I think this deserves a comment :)